server/src/utils/logger.ts

```diff
@@ -0,0 +1,93 @@
+import { createLogger, format, transports } from "winston";
+import dailyRotateFileTransport from "winston-daily-rotate-file";
+import path from "path";
+
+import { env } from "../env";
```

Comment:
```suggestion
import { env } from "@/env";
```

Path alias 써도 좋을거 같아요

Reply:
오 path alias가 설정돼있었다니 너무 좋네요 ㄷㄷ 감사합니다!!